plugin/go.vim

```diff
@@ -129,6 +142,10 @@ augroup vim-go
         autocmd CursorHold *.go nested call go#complete#Info()
     endif
 
+    " Echo the identifier information when completion is done. Useful to see
+    " the signature of a function, etc...
+    autocmd CompleteDone *.go nested call s:echo_go_info()
```

Comment:
Yeah definitely @Shougo :+1: Should it not with one `#` ? Your example has two `##`. If yes why it is that way ?